tests/test_posixDatastore.py

```diff
@@ -0,0 +1,106 @@
+#
+# LSST Data Management System
+#
+# Copyright 2008-2017  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import os
+import unittest
+
+import lsst.utils.tests
+import lsst.afw.table
+
+from lsst.butler.datastore import PosixDatastore
+from lsst.butler.storageClass import SourceCatalog
+
+
+class PosixDatastoreTestCase(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        testDir = os.path.dirname(__file__)
+        catalogPath = os.path.join(testDir, "data", "basic", "source_catalog.fits")
+        self.catalog = lsst.afw.table.SourceCatalog.readFits(catalogPath)
+
+    def _assertCatalogEqual(self, inputCatalog, outputCatalog):
```

Comment:
`inputCatalog` is not used.  I guess you are planning on putting in a more comprehensive equality check in the future?

Reply:
Yes, that was the plan. I'll see if I can find the proper comparisons from where I got the test data.